homeassistant/components/airthings/sensor.py

```diff
@@ -72,6 +78,40 @@
         native_unit_of_measurement=CONCENTRATION_PARTS_PER_BILLION,
         name="VOC",
     ),
+    "light": SensorEntityDescription(
+        key="light",
+        device_class=DEVICE_CLASS_ILLUMINANCE,
+        native_unit_of_measurement=PERCENTAGE,
+        name="Light",
+    ),
+    "virusRisk": SensorEntityDescription(
+        key="virusRisk",
+        native_unit_of_measurement="/ 10",
```

Comment:
What does this unit of measurement mean?

Reply:
Its a one out of ten "score"

Reply:
![image](https://user-images.githubusercontent.com/1519459/134867717-d407ca01-ee01-4362-8308-2774610b6a94.png)

[Documentation for Mold](https://www.airthings.com/what-is-mold)

Reply:
I suggest removing the unit. One out of ten is not really a unit of measurement.

Reply:
Convert it to perecent?

Reply:
Converting to percent could be an option but I'm not sure that will be clear for the user what the percent means. If there's device specific documentation as above its probably better to keep that value (0-10).

Reply:
percent could be confusing as that is the unit of humidity, and the scoring here is something else :)

Reply:
@Danielhiversen Do you still want it to be percentage, or should we keep it without a unit? In case you want percentage, where in the code should the multiplication take place?

Reply:
Should not use percent then.